AdminUI/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Build stage
+FROM node:lts-alpine as build-stage
+WORKDIR /app
+COPY package*.json /app/
+RUN npm install
+COPY ./ /app/
+ARG configuration=production
+RUN npm run build -- --output-path=./dist/out --configuration $configuration
```

Comment:
I'm not familiar with angular, but `/dist/out` seems redundant to me. I would choose either `dist` or `out` (probably `dist`)